python/sglang/srt/two_batch_overlap.py

```diff
@@ -416,18 +466,27 @@ def compute_tbo_children_num_token_non_padded_raw(
 
     @classmethod
     def _compute_split_token_index(cls, batch: ForwardBatch):
+        if batch.forward_mode.is_target_verify():
+            draft_token_num_per_batch = batch.spec_info.draft_token_num
+        else:
+            draft_token_num_per_batch = None
```

Comment:
nit: looks like this logic is repeated several times, maybe we can extract to a short function